package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "webviewer-html-annotate-proxy",
+  "version": "1.0.0",
+  "description": "Annotate live HTML pages by simply providing URL",
+  "scripts": {
+    "server": "npm start --prefix server",
+    "client": "npm start --prefix client",
+    "dev": "concurrently \"npm run client\" \"npm run server\"",
+    "server-dependencies": "npm install --prefix server",
+    "client-dependencies": "npm install --prefix client",
+    "install-all-deps": "concurrently \"npm run server-dependencies\" \"npm run client-dependencies\"",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
The names for the commands are bit confusing to what they do. I have to think about what some of them are doing. The names below more closely match our other repos. `install-all-deps` is fine but we use `setup` in a bunch of other repos. webviewer and pdftron-dev-site come to mind. I prefer staying consistent for ease of use.

 ```
 "scripts": {
    "start-server": "npm start --prefix server",
    "start-client": "npm start --prefix client",
    "start": "concurrently \"npm run client\" \"npm run server\"",
    "install-server": "npm install --prefix server",
    "install-client": "npm install --prefix client",
    "setup": "concurrently \"npm run start-server\" \"npm run start-client\"",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```

Reply:
renamed commands. also added names and colors for concurrently